docs/www/guide-wasm-elastic.md

```diff
@@ -1,261 +1,285 @@
+        remove this Wasm script from the Wasm engine.    
       */
-      const result = new Map();
-      result.set("elasticIndex", recordBatch)
-      /* 
-       Wasm processRecord function has to return a Map always,
-       although that map is empty map
-      */
-      return result
-  })
-  /*
-    Note: if the previous promise failed, the wasm errorHandler is
-    going to handle that error, in this case, Deregister policy will
-    remove this wasm script from wasm engine.    
-  */
-});
+    });
+
+    exports["default"] = transform;
+    ```
 
-exports["default"] = transform;
-```
+6. Bundle the project files into a single file with:
 
-after applying the new logic, we need to bundle everything into a single file.
-For it, we can use 
+    ```bash
+    #this command install all nodejs dependecies
+    $ npm install
+    #this command generate `dist/elastic.js` file.
+    $ npm run build
+    ```
 
-```bash
-#this command install all nodejs dependecies
-$ npm install
-#this command generate `dist/elastic.js` file.
-$ npm run build
-```
+## Deploy the Wasm function to Redpanda
 
-Before deploying this Wasm script, we need to create the `origin` topic
-The statically defined input topic that this Wasm operator has defined in its metadata
+Now we need to deploy the Wasm script so that it can process the transforms.
 
-```bash
-rpk topic create origin
-```
+1. Create a topic to use for the transform:
 
- ## Deploy the Wasm function to Redpanda
+    ```bash
+    rpk topic create origin
+    ```
 
-we use `rpk wasm deploy dist/elastic.js` in this case
+2. Deploy the Wasm transform with:
 
-```bash
-$ rpk wasm deploy dist/elastic.js
+    ```bash
+    rpk wasm deploy dist/elastic.js
+    ```
 
-Sent record to partition 0 at offset 1 with timestamp 2021-02-01 18:18:15.734185538 -0500 -05 m=+0.053943881.
-```
+    The results shows:
 
-To verify that the wasm script has successfully been deployed, 
-you can look through the redpanda logs for a log line that looks like this
-for check it.
+    ```
+    Sent record to partition 0 at offset 1 with timestamp 2021-02-01 18:18:15.734185538 -0500 -05 m=+0.053943881.
+    ```    
 
-on `redpanda logs`
+    To verify that the Wasm script was successfully deployed, 
+    you can run `redpanda logs` to look through the redpanda logs for a log line that looks like:
 
-```bash
-INFO [shard 0] coproc - service.cc:101 - Request recieved to register coprocessor with id: 14103244480447969041
-```
+    ```bash
+    INFO [shard 0] coproc - service.cc:101 - Request recieved to register coprocessor with id: 14103244480447969041
+    ```
 
-The Wasm engine itself also has a log file which is located by default in `/var/lib/redpanda/coprocessor/logs/wasm`. To see the status of your function as reported by the Wasm engine look for log lines like [this]
+    The Wasm engine itself also has a log file that is located by default in `/var/lib/redpanda/coprocessor/logs/wasm`.
```

Comment:
Maybe instead of `/var/lib/redpanda` we should say `RP_DATA_DIR` since its a configurable value

Reply:
hmm, that's ok, that's an additional complexity, think the simplest thing that could possibly work.